lib/zoom/client.rb

```diff
@@ -29,20 +29,32 @@ class Client
     def headers
       {
         'Accept' => 'application/json',
-        'Content-Type' => 'application/json',
+        'Content-Type' => 'application/json'
       }
     end
 
+    def form_url_encoded_headers
+      headers.merge({ 'Content-Type' => 'application/x-www-form-urlencoded' })
```

Comment:
Minor: the curly braces aren't needed.

```suggestion
      headers.merge('Content-Type' => 'application/x-www-form-urlencoded')
```